docs/data/charts/radar/radar.md

```diff
@@ -1,15 +1,71 @@
 ---
 title: React Radar chart
 productId: x-charts
+components: RadarChart, RadarGrid, RadarSeriesArea, RadarSeriesMarks, RadarDataProvider
 ---
 
 # Charts - Radar 🚧
 
 <p class="description">Radar allows to compare multivariate data in a 2D chart.</p>
 
 :::warning
-The Radar Chart component isn't available yet, but you can upvote [**this GitHub issue**](https://github.com/mui/mui-x/issues/7925) to see it arrive sooner.
+🚧 This component is under development 🚧
 
-Don't hesitate to leave a comment there to influence what gets built.
-Especially if you already have a use case for this component, or if you're facing a pain point with your current solution.
+Not all the feature are implemented and its API might change in the future if needed to integrate the upcoming features.
 :::
+
+## Basics
+
+A radar chart is defined by two main props.
+
+- The `series` prop which provides the values to display thanks to the `data` property.
+- The `radar` prop which defines the radar axes.
+
+{{"demo": "BasicRadar.js"}}
+
+## Multi-series
+
+You can plot multiple series on the same radar chart.
+
+{{"demo": "MultiSeriesRadar.js"}}
+
+## Axis
+
+The `metrics` property of `radar` takes an array with one item per corner of the radar.
+This item can either be:
+
+- A string used as the axis label. The other properties will be defaulted.
```

Comment:
```suggestion
- A string used as the axis label. The other properties are populated from the data.
```

I feel that `defaulted` usually has a negative connotation, we should probably not use it in documents. Eg: "John defaulted on his loans"

Another option is to fully remove the second phrase as well, it doesn't add much information.